lib/org/openpsa/invoices/handler/invoice/action.php

```diff
@@ -130,6 +130,17 @@ public function _handler_create_pdf()
         }
     }
 
+    public function _handler_create_pdf_reminder()
+    {
+        $pdf_helper = new org_openpsa_invoices_invoice_pdf($this->invoice);
+        try {
+            $pdf_helper->render_and_attach('reminder');
+            return $this->reply(true, $this->_l10n->get('pdf created'));
```

Comment:
I think this should have spearate texts. I.e. "reminder created", not "pdf created"